scripts/globals/job_utils/dragoon.lua

```diff
@@ -367,7 +367,7 @@ xi.job_utils.dragoon.useSpiritLink = function(player, target, ability)
         local copyi = 0
 
         for _, effect in pairs(effects) do
-            if bit.band(effect:getFlag(), xi.effectFlag.EMPATHY) == xi.effectFlag.EMPATHY then
+            if bit.band(effect:getEffectFlags(), xi.effectFlag.EMPATHY) == xi.effectFlag.EMPATHY then
```

Comment:
I wish I hadn't been busy the day this went in (or hadn't missed it the day I changed those functions). there's actually new functionality this better that skips having to do bitwise and a compare:

```lua
if effect:hasEffectFlag(xi.effectFlag.EMPATHY) then
```
Returns true if it has it, false if it doesn't

not urgent but free easy pr if you want to follow up!